package.json

```diff
@@ -1,41 +1,45 @@
 {
-  "name": "pluribus-project",
-  "version": "0.0.0",
-  "description": "Changing the world with Swarm Imaging!",
-  "engines": {
-    "node": "6.11.1"
-  },
-  "main": "index.js",
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject",
-    "build-css": "node-sass-chokidar src/ -o src/",
-    "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive"
-  },
-  "dependencies": {
-    "ejs": "2.5.6",
-    "express": "4.15.2",
-    "mongodb": "^2.2.30",
-    "node-sass-chokidar": "0.0.3",
-    "npm-run-all": "^4.0.2",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1",
-    "react-scripts": "1.0.10"
-  },
-  "devDependencies": {
-    "request": "^2.81.0",
-    "tape": "^4.7.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": " https://github.com/Pluribus-Technologies/Xylocopa"
-  },
-  "keywords": [
-    "node",
-    "heroku",
-    "express"
-  ],
-  "license": "MIT"
+    "name": "pluribus-project",
+    "version": "0.0.0",
+    "description": "Changing the world with Swarm Imaging!",
+    "engines": {
+        "node": "6.11.1"
+    },
+    "main": "index.js",
+    "scripts": {
```

Comment:
All of the below Windows-specific batch scrips (*.bat) should actually be here so they can be invoked via `npm {command}` in a shell.